src/pages/index.astro

```diff
@@ -1,127 +1,226 @@
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 1.5;
+    padding-bottom: 40px;
+    color: #4a4a4a;
+  }
+
+  .small-body {
+    font-size: 18px;
+    font-weight: 400;
+    line-height: 1.5;
+    margin-left: 16px;
+    color: #4a4a4a;
+  }
+
+  .small-body-decoration {
+    border-left: #0c56bc 2px solid;
+    margin: 24px 0px;
+  }
+
+  nav a {
+    color: #0c56bc;
+    font-weight: 500;
+    text-decoration: none;
+    position: relative;
+  }
+
+  nav .main-menu a::after {
+    content: '';
+    position: absolute;
+    background-color: #0c56bc;
+
+    left: 0;
+    bottom: -4px;
+
+    height: 1.5px;
+    width: 24px;
+
+    transition-property: width;
+    transition-duration: 0.3s;
+    transition-timing-function: ease-out;
+  }
+  nav a:hover {
+    color: #00308f;
+  }
+
+  nav a:hover::after {
+    width: 100%;
+    color: #00308f;
+    background-color: #00308f;
+  }
+
+  /* Języki */
+
+  nav .lang a::after {
+    content: '';
+    position: absolute;
+    background-color: #0c56bc;
+
+    left: 0;
+    bottom: -4px;
+
+    height: 1.5px;
+    width: 50%;
+
+    transition-property: width;
+    transition-duration: 0.3s;
+    transition-timing-function: ease-out;
+  }
+
+  nav a:hover {
+    color: #00308f;
+  }
+
+  nav a:hover::after {
+    width: 100%;
+    color: #00308f;
+    background-color: #00308f;
+  }
+
+  /* Wybrany język */
+
+  nav .choosen a {
+    color: #000;
+  }
+
+  nav .choosen a::after {
+    background-color: #000;
+
+    width: 100%;
+  }
+
+  nav label {
+    display: none;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  /* Navigation */
+
+  .main-nav {
+    align-items: center;
+    padding-top: 40px;
+  }
+
+  .main-nav .logo {
+    height: 24px;
+    margin-bottom: 24px;
+  }
+
+  .main-nav ul {
+    display: flex;
+    gap: 24px;
+  }
+
+  .main-nav ul li {
+    display: flex;
+    gap: 24px;
+  }
+
+  .main-nav ul li a {
+    margin: 8px 0px;
+  }
+
+  /* Header */
+
+  header {
+    width: 100%;
+    height: 75vh;
+    align-items: center;
+    justify-content: baseline;
+    background-size: cover;
+    background-position-x: center;
+    background-position-y: center;
+  }
+  header img {
+    width: 100%;
+    opacity: 70%;
+    object-fit: cover;
+  }
+
+  .header-section {
+    flex: 1;
+    max-width: 50%;
+  }
+
+  .header-section h1 {
+    margin: 16px 0px;
+  }
+
+  .header-section a {
+    text-decoration: none;
+    font-weight: 500;
+    display: inline-block;
+    color: #0c56bc;
+    transition: ease-in;
+  }
+
+  .header-section a:hover {
+    color: #00308f;
+  }
+
+  .header-section a i {
+    transition: all 0.2s ease-in-out;
+  }
+
+  .header-section a:hover i {
+    color: #00308f;
+    margin-left: 8px;
+  }
+
+  /* About */
+
+  .about-section {
+    text-align: left;
+  }
+
+  .about-section .about-information {
+    margin: 104px 0px;
+  }
+
+  .about-section .img-about-container {
+    max-width: min(960px, 100%);
+    height: 100%;
+    object-fit: cover;
   }
```

Comment:
unused styles

Reply:
it is true. I removed it at this moment.